.travis.yml

```diff
@@ -11,4 +12,5 @@ go:
 env:
   - TARGET=docker
```

Comment:
I thought the intention was to use
```
env:
 - TARGET=docker
 - TARGET=run
 - TARGET=test
script: make $TARGET
```
so that travis creates a matrix of jobs. It won't work in your case since all targets depend on each other though.

Reply:
I thought the mention was to maybe have a version that worked with podman.

In your example, `make run` wouldn't see the results from `make docker`, the jobs in a matrix run concurrently don't that?

I was thinking it'd be more like TARGET=docker and TARGET=podman to test on both (if travis ever supported both)